scripts/build_awslc.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
```

Comment:
```suggestion
#!/bin/sh
set -e
set -o xtrace
```

Enable trace logging of the shell script, and fail on error. This helps debug CI failure.

Reply:
Done